lib/node.js

```diff
@@ -2216,6 +2216,26 @@ declare module "zlib" {
 
 declare module "assert" {
   declare class AssertionError extends Error {}
+  declare type AssertStrict = {
+    (value: any, message?: string): void;
+    ok(value: any, message?: string): void;
+    fail(message?: string | Error): void;
+    // deprecated since v10.15
+    fail(actual: any, expected: any, message: string, operator: string): void;
```

Comment:
Should we update `assert.fail` the same way it's done there https://github.com/facebook/flow/pull/7915?